src/Microsoft.SqlTools.ServiceLayer/LanguageServices/AutoCompleteHelper.cs

```diff
@@ -343,6 +343,27 @@ internal static CompletionItem[] EmptyCompletionList
         }
 
         /// <summary>
+        /// Retrieves the reserved word list 
+        /// </summary>
+        internal static string[] GetReservedWordList
```

Comment:
> internal static string[] GetReservedWordList [](start = 7, length = 45)

this property doesn't appear to be used anywhere now since IsReserveredWord was moved into this class...if it's no longer needed can we delete it?